src/server/config.js

```diff
@@ -1,48 +1,47 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 
-var config = {};
+/**
+ * The Configuration model manages the configuration properties for the simulation.
+ * These properties are mapped with the options provided for the simulation, and
+ * are cross to any other instance involved in the simulation.
+ * @constructor
+ */
+function Configuration() {
+    this._config = {};
+}
```

Comment:
Maybe it would be nicer to not inherit from `Object`, but not have a prototype at all:

```javascript
Configuration.prototype = Object.create(null);
Configuration.prototype.constructor = Configuration;
```